### ISSUE TITLE
Unable to reproduce paper metrics, questions about paper results

### ISSUE THREAD

**Issue by wjaycyf** [KAIST-VICLab/FMA-Net#23]
Hi, I downloaded the train_blur_bicubic and val_blur_bicubic datasets of type Blur + Low Resolution according to the dataset link you provided，
![image](https://github.com/user-attachments/assets/30fda1dc-2731-497d-bf89-0a3b8e4ab493)
Next, I run . /preprocessing/generate_reds4.py and . /preprocessing/generate_flow.py, which generated the corresponding reds4 validation set and the corresponding optical flow file as follows，
![image](https://github.com/user-attachments/assets/d61740fc-5dc6-4e95-afe8-0007abf24f30)
Using the above files as a test set, gt is the corresponding files taken from the dataset links train_orig_part0.zip and train_orig_part1.zip. 
Next, I run python main.py --test --config_path experiment.cfg with the provided pre-training weights and the accuracy results of the test are as follows，
![image](https://github.com/user-attachments/assets/d5492882-fb1d-4e7f-a545-6ce6379220ee)
![image](https://github.com/user-attachments/assets/47bc60c6-e475-47f1-89f9-82d84718798b)
![image](https://github.com/user-attachments/assets/f56024bb-9fc5-4ae3-92b5-3c34ef432078)
The results are quite different from the results of your paper, may I ask which type of data from REDS did you use for the experiment? If I want to reproduce the results of your paper, which type of dataset do I need to download? Looking forward to your reply.

**Comment by GeunhyukYouk** [comment]
Hello,
It seems that you have downloaded the wrong GT data.
The GT data should be train_sharp and val_sharp, not train_origin. Please refer to the image below.
![image](https://github.com/user-attachments/assets/a191d6f8-be8a-444d-8ab3-9176ea316d39)

Also, generate_flow.py is required only for generating pseudo-GT optical flow used in **training**, so it is not necessary when testing with the **provided pretrained model**.
